docs-content/index.html.md.erb

```diff
@@ -75,40 +74,122 @@ The following table provides version and version-support information about Cyber
+        <tr>
+            <td>
+                **URL of the Conjur follower load balancer**<br/>
+                (Conjur Enterprise v5.x+)<br/>
+                (Optional)
+            </td>
+            <td>
+                If deploying a cluster in an HA configuration, you can specify a URL for PCF apps to connect to
+                a Conjur follower to retrieve secret values rather than through the Conjur master. Cyberark strongly
+                recommends that you supply a load balanced follower address to support easier scalability.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                **x509 CA Certificate Chain for Conjur**<br/>
+                (Conjur EE v5.x+ and Conjur OSS v1.x+)<br/>
+                (Optional)
+            </td>
+            <td>
+                The PEM encoded x509 certificate chain for the Conjur server. The certificate is
+                stored in a `.pem` file that was created by the `conjur init` command when using the
+                Conjur CLI. The default path for this file is `/conjur_install_directory/conjur-account.pem`.
+            </td>
+        </tr>
+        <tr>
+        </tr>
+    </table>
+
+
++ **The Conjur Service Broker requires that each space have its own service instance.**
+
+    Conjur Service instances may not be [shared across spaces](https://docs.cloudfoundry.org/devguide/services/sharing-instances.html).
+    In the interest of least privilege, a single Conjur service instance must be 
+    provisioned in each space where it is needed, using:
+
+    <pre class='term'>$ cf create-service cyberark-conjur community my-service-instance-name</pre>
+
++ **(Conjur EE v4.x only) Conjur Host Factory**
+
+    The Service Broker requires a Conjur host factory to use when adding host identities. The examples in [Recommendations](#recommendations) include host factory definitions. If a host factory is not properly defined, a health check near the end of tile installation will report the condition, and the tile installation will fail.
 
 
 ## <a id="recommendations"></a>Recommendations
 
 ### Create a Dedicated Conjur Policy for PCF
 
 CyberArk strongly recommends that you create a Conjur policy dedicated to PCF before installing the PCF tile.
+This policy should include a unique identifier for the foundation (e.g. `production`) to make managing Conjur
+policy easier across multiple PCF installation.
 
-The syntax for defining a Conjur Policy with ID `pcf` is as follows:
+The syntax for defining a Conjur Policy with ID `pcf/production` is as follows:
 
 ```
 - !policy
   id: pcf
+  body:
+  - !policy
+    id: production
 ```
 
-This can be loaded into the root policy or any other Conjur policy you would like. Host identities for PCF-deployed applications will be loaded into the `pcf` Conjur policy when the applications are bound to a Conjur service instance.
+This can be loaded into the root policy or any other Conjur policy you would like. Host identities for 
```

Comment:
"or any other Conjur policy branch you would like" - I strongly prefer adding the `branch`, otherwise `policy` is an overloaded term